content/blog/2024-08-08-tmix-07/index.md

```diff
@@ -31,7 +40,7 @@ Widgets are now stored in a global arena, which has little impact on the public
 
 - [xilem#417][] exposes an unstable API for embedding Masonry and Xilem into an existing Winit application.
 - [xilem#418][], [xilem#421][] improved our unofficial support for iOS.
-- [rfcs#7][] specifies a clearer architecture for passes, including simplifications necessary for compositor use.
+- [rfcs#7][] specifies a clearer architect  ure for passes, including simplifications necessary for compositor use.
```

Comment:
```suggestion
- [rfcs#7][] specifies a clearer architecture for passes, including simplifications necessary for compositor use.
```

Reply:
This looks like a duplicate of de71af3ffa12f8fbeaa60430785aa2f40f9e19cf?